lexicons/com/atproto/temp/addReservedHandle.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.temp.addReservedHandle",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Add a handle to the set of reserved handles.",
+      "input": {
+        "encoding": "application/json",
```

Comment:
I recommend having an `output` as well, even if its schema has no fields in it.  This makes it easier to adapt the endpoint in a backwards-compatible way if it requires some output in the future.  For this reason we've started to establish it as a lexicon best practice.